thesis-gui/src/components/my-quizzes/quiz-builder/QuizBuilder.tsx

```diff
@@ -33,7 +33,7 @@ export const QuizBuilder: React.FC<{
     );
 
     const currentQuestion = useMemo(
-        //if update of currentQuestionNumber happened before creating new question object, fallback to last question
+        //if update of currentQuestionNumber happened before creating new ranked-quiz object, fallback to last ranked-quiz
```

Comment:
Why change this? I meant `QuestionForm` object, so `question` seems to fit better than `ranked-quiz`. You can rename this to `QuestionForm object` if you want.

Reply:
Sorry, this must have been changed when I changed component name. I will revert it, sorry!

Reply:
Done